services/homebrew/homebrew-downloads.service.js

```diff
@@ -0,0 +1,82 @@
+'use strict'
+
+const Joi = require('joi')
+const { downloadCount } = require('../color-formatters')
+const { metric } = require('../text-formatters')
+const { BaseJsonService } = require('..')
+const { nonNegativeInteger } = require('../validators')
+
+function getSchema({ formula }) {
+  return Joi.object({
+    analytics: Joi.object({
+      install: Joi.object({
+        '30d': Joi.object({ [formula]: nonNegativeInteger }).required(),
+        '90d': Joi.object({ [formula]: nonNegativeInteger }).required(),
+        '365d': Joi.object({ [formula]: nonNegativeInteger }).required(),
+      }).required(),
+      install_on_request: Joi.object({
```

Comment:
If we're only using the `installs` value we don't need to validate `install_on_request`. Lets remove this. If we add a badge that uses it later, we can add it to the schema then.

Reply:
Okay sounds good!

Reply:
Resolved in https://github.com/badges/shields/pull/6209/commits/9028a67c0264d914c4fd310014bdb7a28c77dd2c